cocotb/__init__.py

```diff
@@ -103,16 +100,24 @@ def _reopen_stream_with_buffering(stream_name):
     if not sys.warnoptions:
         warnings.simplefilter("default")
```

Comment:
This doesn't really belong in a function called `setup_logging`. Perhaps it should move to the regression manager.

Reply:
It doesn't really have much to do with regression, but I will move it into `_initialize_testbench`